addons/binding/org.openhab.binding.minecraft/about.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
```

Comment:
Note that we have a new template for the about.html, see e.g. https://github.com/openhab/openhab2-addons/blob/master/addons/ui/org.openhab.ui.cometvisu.php/about.html.